webservices/docs.py

```diff
@@ -1194,3 +1194,37 @@ def add_ytd(var):
 LINK_TO_REPORT = '''
 URL for retrieving the PDF document
 '''
+
+SUB_ID = '''
+A unique identifier of the transactional report.
+'''
+
+STATUS_NUM = '''
```

Comment:
If we only allow users to see `status_num` 0 and 1, we shouldn't let them choose 4 and 9 in the dropdown.

Reply:
removed the 4 and 9 text from the docs.